analysed-packages/jszip-utils/version-v0.0.2/README.md

```diff
@@ -0,0 +1,23 @@
+## Download Location
+
+https://github.com/Stuk/jszip-utils/archive/refs/tags/v0.0.2.zip
+
+## Package URL (purl)
+
+pkg:github/stuk/jszip-utils@0.0.2
+
+## Creator
+
+Dorothea Kraus
+
+## Reviewers
+
+The information was reviewed by:
+
+* Parian Golchin
+* Oliver Fendt
+
+## Comment
```

Comment:
Since it is main license of the file, you should not write a comment 
`PackageLicenseDeclared: LicenseRef-GPL-3.0-only OR LicenseRef-MIT-a09b4c8ba47c9bec1b147e94aa512f41`

Reply:
I thought this comment was needed whenever the package license was a dual license. What do you mean by "main license of this file"?

Reply:
This has been solved. Based on our newest knowledge (written conversation between Parian and Oliver), only the dual package license has to be mentioned in the README.